test/unit/SubscriberSpec.scala

```diff
@@ -2,9 +2,7 @@ import org.specs2.mutable._
 import play.api.test._
 import play.api.test.Helpers._
 
-import models.Channel
-import models.Collection
-import models.ContentType
+import models.{Agent, Channel, Collection, ContentType}
```

Comment:
Ah... I'll start doing this too. Listing one per line was annoying me as I was doing it :)